tests/common/mod.rs

```diff
@@ -63,7 +167,7 @@ async fn smoke_test() {
     assert_eq!(db.name(), "bar");
     assert_eq!(db.version(), 1);
     assert_eq!(db.object_store_names(), &["objects", "stuffs", "things"]);
-    db.close();
+    drop(db); // Close the database
```

Comment:
Can we add `fn close(self) {}` to `OwnedDatabase` just for consistency with `Database`?